cfgen/examples/ldr1-cluster.yaml

```diff
@@ -35,6 +35,7 @@ nodes:
     m0_clients:
         s3: 0
         other: 2
+create_aux: false # optional; supported values: "false" (default), "true"
```

Comment:
If it's optional then probably it can be omitted or commented out?

Reply:
Removed.